tests/queries/0_stateless/02240_system_remote_filesystem_cache.reference

```diff
@@ -7,8 +7,6 @@ CREATE TABLE test (key UInt32, value String) Engine=MergeTree() ORDER BY key SET
 INSERT INTO test SELECT number, toString(number) FROM numbers(100);
 SELECT  * FROM test FORMAT Null;
 SELECT file_segment_range_begin, file_segment_range_end, size FROM system.filesystem_cache ORDER BY file_segment_range_end, size;
-0	0	1
-0	79	80
```

Comment:
It is very strange here. Why is there a segment of [0, 1) and [0, 79). They don't seem to appear in my commit output. Can you please explain the reason for this difference or this output is indeterminate output？ @kssenii